v0/destinations/bqstream/transform.js

```diff
@@ -51,59 +23,20 @@ async function process(event) {
   }
   const {
     destination: {
-      Config: { datasetId, tableId, projectId }
-    },
-    destination
+      Config: { datasetId, tableId, projectId, insertId }
```

Comment:
can we rename `insertId` to `insertIdColumn` or something, just for clarity?

Reply:
Yeah. Will change the naming